tools/create_audio_messages.py

```diff
@@ -51,7 +51,5 @@
             match = lineRe.match(line.strip())
             if match:
                 fileName = match.group(1)
-                if args.only_new and os.path.isfile(targetDir + "/" + fileName):
-                    continue
                 text = match.group(2)
-                text_to_speech.textToSpeechUsingArgs(text=text, targetFile=targetDir + "/" + fileName, args=args)
+                text_to_speech.textToSpeechUsingArgs(text=text, targetFile=targetDir + '/mp3/' + fileName, args=args)
```

Comment:
I think this line should stay unchanged. The "mp3" and "advert" sub-folder get added automatically. No need to manually add "mp3" here.

Reply:
The folder gets added yes. But this is to place the files in the correct folder or?

Reply:
no, the filename already contains the folder ("mp3" or "advert").
The targetDir can be specified using the "output" argument. For example:
python3 tools/create_audio_messages.py --output=sd-card --use-google-key=...

Reply:
I'm no python expert, but you are right for the generation of the numbers. But for the regular files, and that's what this line is about, it add 'mp3' so they are placed in the mp3 folder.

Reply:
And the mp3 folder is within the folder specified by --output.